macos/snd_.ksy

```diff
@@ -0,0 +1,201 @@
+meta:
+  id: snd_
+  endian: be
+  xref:
+    wikidata: Q7564684
+    reference: "https://developer.apple.com/library/archive/documentation/mac/pdf/Sound/Sound_Manager.pdf"
+enums:
+  data_type:
+    0x01: square_wave_synth
+    0x03: wave_table_synth
+    0x05: sampled_synth
+  cmd_type:
+    00: null_cmd
+    03: quiet_cmd
+    04: flush_cmd
+    10: wait_cmd
+    11: pause_cmd
+    12: resume_cmd
+    13: call_back_cmd
+    14: sync_cmd
+    15: empty_cmd
+    40: freq_duration_cmd
+    41: rest_cmd
+    42: freq_cmd
+    43: amp_cmd
+    44: timbre_cmd
+    60: wave_table_cmd
+    61: phase_cmd
+    80: sound_cmd
+    81: buffer_cmd
+    82: rate_cmd
+  sound_header_type:
+    0x00: standard
+    0xFF: extended
+    0xFE: compressed
+seq:
+  - id: format
+    type: u2
+  - id: number_of_data_formats
+    type: u2
+    if: format==1
+  - id: data_format
+    type: data_format
+    repeat: expr
+    repeat-expr: number_of_data_formats
+    if: format==1
+  - id: reference_count
+    type: u2
+    if: format==2
+  - id: number_of_sound_commands
+    type: u2
+  - id: sound_command
+    type: sound_command
+    repeat: expr
+    repeat-expr: number_of_sound_commands
+types:
+  data_format:
+    seq:
+      - id: id
+        type: u2
+        enum: data_type
+      - id: options
+        type: u4
+  sound_command:
+    instances:
```

Comment:
According to the style guide, the `instances` section should come after `seq`.

Reply:
Done.